CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [Stable release 5.2.1] - 2024-05-28
+
+This update brings the below changes over the previous release:
+
+### Changed
+
+- Upgraded `Azure.Identity` version from 1.10.3 to 1.11.3 [#2528](https://github.com/dotnet/SqlClient/pull/2528)
+- Code Health improvements: [#2467](https://github.com/dotnet/SqlClient/pull/2467)
```

Comment:
I think this needs to be moved under Fixed section. This is my opinion. What do you think @DavoudEshtehari

Reply:
This fix hasn't changed the functionality, and I believe it's more appropriate under the current title.

Reply:
This has more changes over upgrading Azure.Identity that I'd mention them here as well!